### DISCUSSION TITLE
How should I choose appropiates []Q and []P for a specific LogN

### DISCUSSION THREAD

**Discussion by fedejinich** [tuneinsight/lattigo#365] [Q&A]
I want to do an experiment with `Log N = 16`, how should I choose the right []Q and []P?

**Comment by Pro7ech** [comment]
Giving `Q := []uint64{}` and `P := []uint64` isn't necessary. This is only if you want to give specific moduli. Else you can give `LogQ := []int{}` and `LogP := []int{}`.
